ui/desktop/app/templates/scopes/scope/projects.hbs

```diff
@@ -15,6 +15,10 @@
         <Rose::Icon @name='flight-icons/svg/entry-point-16' @size='medium' />
         {{t 'resources.session.title_plural'}}
       </nav.link>
+      <nav.link @route='scopes.scope.projects.settings'>
+        <Rose::Icon @name='flight-icons/svg/settings-16' @size='medium' />
+        {{t 'titles.settings'}}
+      </nav.link>
```

Comment:
Sorry if stated before but do we need a feature flag for this in-progress work?

Reply:
since we are merging this to a LLB I don't think we need one

Reply:
Gotcha, thanks. I saw main as the base branch. Good with this as is going into an llb.